tests/s3/copy_object/test_parallel_copy_object.py

```diff
@@ -116,6 +117,21 @@ def parallel_copy_and_put_object(self, copy_args, put_args):
             assert_utils.assert_true(process1.get()[0], process1.get()[1])
             assert_utils.assert_true(process2.get()[0], process2.get()[1])
 
+    def multi_parallel_copy_object(self, *args, numproc=2, assert_flag=True):
+        """Parallel copy object and put object with given arguments"""
+        pool = multiprocessing.Pool()
+        process = []
+        for i in range(numproc):
+            LOGGER.info("Parallely, \n1. From %s copy %s to %s as %s",
+                        args[i][0], args[i][1], args[i][2], args[i][3])
+        for i in range(numproc):
+            process.append(pool.apply_async(self.copy_object_wrapper, args=args[i]))
+        pool.close()
+        pool.join()
+        if assert_flag:
+            for i in range(numproc):
+                assert_utils.assert_true(process[i].get()[0], process[i].get()[1])
+
```

Comment:
should we not handle assert_flag=False here , with assert_false?

Reply:
assert_flag=False we are capturing exception in tests